charts/navi-back/templates/service.yaml

```
@@ -9,21 +10,76 @@ metadata:
     {{- end }}
     rule: {{ .Values.naviback.app_rule | default "" | quote  }}
     navigroup: {{ .Values.navigroup | default "" | quote }}
-    type: {{ .Values.naviback.type | quote  }}
+    type: {{ .Values.naviback.type | default "" | quote  }}
+    {{- if .Values.dataGroup.enabled }}
+    dataGroupTimestamp: {{ .Values.dataGroup.timestamp | quote }}
+    dataGroupPrefix: {{ .Values.dataGroup.prefix | quote }}
+    {{- end }}
     {{- if .Values.naviback.behindSplitter }}
     behindSplitter: "true"
     {{- end }}
   {{- if .Values.service.annotations }}
   annotations:
    {{- include "tplvalues.render" ( dict "value" .Values.service.annotations "context" $) | nindent 4 }}
```

3 пробела, добавил еще один
```suggestion
    {{- include "tplvalues.render" ( dict "value" .Values.service.annotations "context" $) | nindent 4 }}
```